ntbs-integration-tests/NotificationPages/OverviewPageTests.cs

```diff
@@ -134,7 +156,26 @@ public async Task OverviewPageShowsCreateNewNotificationForThisPatientButton_IfN
             // Arrange
             var url = GetCurrentPathForId(Utilities.NOTIFICATION_DATE_OVER_YEAR_AGO);
             var document = await GetDocumentForUrl(url);
+            
+            // Assert
             Assert.NotNull(document.QuerySelector("#new-linked-notification-button"));
         }
+        
+        [Fact]
+        public async Task OverviewPageShowsReadOnlyVersion_IfReadOnlyPermission()
+        {
+            using (var client = Factory.WithMockUserService<TestNhsUserService>()
+                .CreateClientWithoutRedirects())
+            {
+                // Arrange
+                var url = GetCurrentPathForId(Utilities.LINK_NOTIFICATION_ROYAL_FREE_LONDON_TB_SERVICE);
+                var response = await client.GetAsync(url);
+                var document = await GetDocumentAsync(response);

```

Comment:
I think this is actually the `Act` part - the thing you're testing here is what you can get on that page after all!

This brings me to the fact that the arrange is slightly *tricky* in this case - the arranging is setting up the test so that the user has read only access to the notification. Currently, I believe this is being done through `using (var client = Factory.WithMockUserService<TestNhsUserService>().CreateClientWithoutRedirects())`, but nothing about this code suggests any relation to LINK_NOTIFICATION_ROYAL_FREE_LONDON_TB_SERVICE. Perhaps we should give `TestNhsUserService` a different name, one that indicates that it mocks in a user from `TBSERVICE_ABINGDON_COMMUNITY_HOSPITAL_ID` and `PERMITTED_SERVICE_CODE`? (after https://github.com/PublicHealthEngland/ntbs_Beta/blob/421bbd2d1e76bef594c494dc673d2db6cbd67d63/ntbs-integration-tests/TestServices/TestUserService.cs#L21-L25)
(While here - the name `PERMITTED_SERVICE_CODE` also seems very difficult to pin down in the abstract, maybe worth making a bit more concrete!)

The other bit of arranging is actually the seeding of the data - perhaps that's worth commenting on in the test?